### ISSUE TITLE
Too many trackables cause crash in importing geocache from GC code or from link

### ISSUE THREAD

**Issue by arcao** [arcao/Geocaching4Locus#61]
From: http://help.locusmap.eu/topic/locus-crashes-when-clicking-in-mega-event-cache-in-live-map

When you try to import geocache from GC code or form link which has more than 100 trackables, Locus Map crash on start when tries to unpackage the Intent data (Geocache data are sent via Intent).

Example cache: GC54932

Log:
```
06-21 23:17:54.990      471-973/system_process I/ActivityManager﹕ START u0 {act=locus.api.android.ACTION_DISPLAY_DATA flg=0x10000000 cmp=menion.android.locus.pro/com.asamm.locus.basic.MainActivityBasic (has extras)} from uid 10072 on display 0
06-21 23:17:54.994      471-973/system_process V/WindowManager﹕ addAppToken: AppWindowToken{28f5287e token=Token{186a4a39 ActivityRecord{11ba4200 u0 menion.android.locus.pro/com.asamm.locus.basic.MainActivityBasic t151}}} to stack=1 task=151 at 0
06-21 23:17:54.999      471-494/system_process V/WindowManager﹕ Adding window Window{39c3c956 u0 Starting menion.android.locus.pro} at 6 of 11 (after Window{2789b083 u0 com.arcao.geocaching4locus/com.arcao.geocaching4locus.ImportFromGCActivity})
06-21 23:17:55.056      198-198/? E/EGL_emulation﹕ tid 198: eglCreateSyncKHR(1209): error 0x3004 (EGL_BAD_ATTRIBUTE)
06-21 23:17:55.147    4731-4731/com.arcao.geocaching4locus D/ImportFromGCActivity﹕ onImportFinished result: true
06-21 23:17:55.161      471-991/system_process E/JavaBinder﹕ !!! FAILED BINDER TRANSACTION !!!
06-21 23:17:55.173      471-991/system_process W/ActivityManager﹕ Exception when starting activity menion.android.locus.pro/com.asamm.locus.basic.MainActivityBasic
    android.os.TransactionTooLargeException
            at android.os.BinderProxy.transactNative(Native Method)
            at android.os.BinderProxy.transact(Binder.java:496)
            at android.app.ApplicationThreadProxy.scheduleLaunchActivity(ApplicationThreadNative.java:797)
            at com.android.server.am.ActivityStackSupervisor.realStartActivityLocked(ActivityStackSupervisor.java:1181)
            at com.android.server.am.ActivityStackSupervisor.startSpecificActivityLocked(ActivityStackSupervisor.java:1281)
            at com.android.server.am.ActivityStack.resumeTopActivityInnerLocked(ActivityStack.java:1891)
            at com.android.server.am.ActivityStack.resumeTopActivityLocked(ActivityStack.java:1455)
            at com.android.server.am.ActivityStackSupervisor.resumeTopActivitiesLocked(ActivityStackSupervisor.java:2473)
            at com.android.server.am.ActivityStack.completePauseLocked(ActivityStack.java:998)
            at com.android.server.am.ActivityStack.activityPausedLocked(ActivityStack.java:896)
            at com.android.server.am.ActivityManagerService.activityPaused(ActivityManagerService.java:6311)
            at android.app.ActivityManagerNative.onTransact(ActivityManagerNative.java:512)
            at com.android.server.am.ActivityManagerService.onTransact(ActivityManagerService.java:2208)
            at android.os.Binder.execTransact(Binder.java:446)
06-21 23:17:55.264      471-991/system_process I/ActivityManager﹕ Start proc 5181:menion.android.locus.pro/u0a73 for activity menion.android.locus.pro/com.asamm.locus.basic.MainActivityBasic
06-21 23:17:55.266      471-824/system_process W/ActivityManager﹕ Spurious death for ProcessRecord{1370446d 5181:menion.android.locus.pro/u0a73}, curProc for 4285: null
06-21 23:17:55.488      471-484/system_process E/JavaBinder﹕ !!! FAILED BINDER TRANSACTION !!!
...
06-21 23:17:55.495      471-484/system_process I/ActivityManager﹕ Killing 5181:menion.android.locus.pro/u0a73 (adj 0): error during init
```

**Comment by arcao** [comment]
Reopen: Update geocache from simple data to full data cause crash too.

**Comment by arcao** [comment]
Fixed by hack in: 9279287b8563525ec641c3576f77676005d0d931

**Comment by arcao** [comment]
Fixed.
